### ISSUE TITLE
how to click mouse move?

### ISSUE THREAD

**Issue by YeonwooJeong** [microsoft/WinAppDriver#884]
I know located element

How found:	Mouse move (801,431)
	hwnd=0x00110C34 64bit class="#32768" style=0x94800000 ex=0x189

how to mouse move click?

I want to use location number.

but Related api is deprecated
![image](https://user-images.githubusercontent.com/8816386/65478814-1e044d00-dec6-11e9-8be9-e1874effdb55.png)

**Comment by micheal650** [comment]
For example with C#

driver.Mouse.MouseMove(windowsElement.Coordinates,x,y);
driver.Mouse.Click(null);

**Comment by YeonwooJeong** [comment]
> For example with C#
> 
> driver.Mouse.MouseMove(windowsElement.Coordinates,x,y);
> driver.Mouse.Click(null);

Can't I do it without an windowsElement?
Element is not supported.

**Comment by KitoCoding** [comment]
Yes you can, are you using a framework like appium? or selenium?

**Comment by YeonwooJeong** [comment]
> Yes you can, are you using a framework like appium? or selenium?

like Selenium

**Comment by KitoCoding** [comment]
Selenium doesn't have direct capabilities for winappdriver. Ohhh wait your using java. I can tell you how to in c# but java I have no idea if it's possible

**Comment by YeonwooJeong** [comment]
> Selenium doesn't have direct capabilities for winappdriver. Ohhh wait your using java. I can tell you how to in c# but java I have no idea if it's possible

yes. I using java. therefore Is it available in appium?  Thanks for your answer.
